extensions/interactions/AlgebraicExpressionInput/directives/algebraic-expression-input-validation.service.ts

```diff
@@ -39,6 +39,8 @@ import { NumericExpressionInputRulesService } from 'interactions/NumericExpressi
   providedIn: 'root'
 })
 export class AlgebraicExpressionInputValidationService {
+  private supportedFunctionNames = AppConstants.SUPPORTED_FUNCTION_NAMES;
```

Comment:
Why is this needed?

Reply:
This is used to show the list of supported functions via the validation error message.